lib/improver/tests/utilities/test_OrographicAlphas.py

```diff
@@ -0,0 +1,188 @@
+               'invert_alphas: {}>'.format(0.0, 1.0, 1, 1, True))
+        self.assertEqual(result, msg)
+
+
+class Test_scale_alphas(IrisTest):
+    """Class to test the scale_alphas function"""
+
+    def setUp(self):
+        """Set up cube & plugin"""
+        self.plugin = OrographicAlphas()
+        cube = set_up_cube()
+        self.cubelist = [cube, cube]
+
+    def test_basic(self):
+        """
+        Test the basic function of scale_alphas, using the
+        standard max and min alphas.
+        """
+        result = self.plugin.scale_alphas(self.cubelist)
+        expected = np.array([[0.1, 0.5, 1.0],
+                             [0.3, 0.4, 0.7],
+                             [0.0, 0.2, 0.1]])
+        self.assertArrayAlmostEqual(result[0].data, expected)
+        self.assertArrayAlmostEqual(result[1].data, expected)
+
+    def test_maxmin(self):
+        """
+        Tests the function of scale_alphas, using a max
+        and min value for alpha.
+        """
+        result = self.plugin.scale_alphas(self.cubelist, 0.3, 0.5)
+        expected = np.array([[0.32, 0.40, 0.50],
+                             [0.36, 0.38, 0.44],
+                             [0.30, 0.34, 0.32]])
+        self.assertArrayAlmostEqual(result[0].data, expected)
+        self.assertArrayAlmostEqual(result[1].data, expected)
+
+
+class Test_unnormalised_alphas(IrisTest):
+    """Dummy class to test the basic alphas function (currently no effect)"""
+
+    def setUp(self):
+        """Set up cube & plugin"""
+        self.plugin = OrographicAlphas()
+        self.cube = set_up_cube()
+
+    def test_basic(self):
+        """Test data are as expected"""
+        gradient_x, _ = \
+            DifferenceBetweenAdjacentGridSquares(gradient=True).process(
+                self.cube)
+        alpha_x = self.plugin.unnormalised_alphas(gradient_x)
+        self.assertArrayAlmostEqual(gradient_x.data, alpha_x.data)
+
+
+class Test_gradient_to_alpha(IrisTest):
+
+    """Class to test alphas data and metadata output"""
+
+    def setUp(self):
+        """Set up cube & plugin"""
+        self.plugin = OrographicAlphas(min_alpha=0.3, max_alpha=0.5)
+        self.cube = set_up_cube()
+        self.gradient_x, self.gradient_y = \
+            DifferenceBetweenAdjacentGridSquares(gradient=True).process(
+                self.cube)
+
+    def test_basic(self):
+        """Test basic version of gradient to alpha"""
+        expected = np.array([[0.32, 0.40, 0.50],
+                             [0.36, 0.38, 0.44],
+                             [0.30, 0.34, 0.32]])
+        result = self.plugin.gradient_to_alpha(self.gradient_x,
+                                               self.gradient_y)
+        self.assertEqual(result[0].name(), 'alphas')
+        self.assertArrayAlmostEqual(result[0].data, expected)
+        self.assertNotIn('forecast_period', result[0].coords().name)
+        self.assertNotIn('forecast_time', result[0].coords().name)
```

Comment:
```result[0].coords().name``` I don't think this is a thing?
Usually we can generate a list of coordinate names with something like this:
```[coord.name() for coord in cube.coords()]```
but for this test you could probably use:
```assertFalse(result[0].coords('forecast_period'))``` which will return an empty list if the coordinate is not present.